<launch>

  <node name="kinectCalibration" pkg="detection" type="kinectCalibration" args="kinect2_xavier_link" respawn="false" output="screen">
  </node> 
  <node name="joy_node" pkg="joy" type="joy_node" args="" respawn="false" output="screen">
  </node>

</launch>